_includes/banner.html

```diff
@@ -1,10 +1,11 @@
 <div class="row banner">
-    <div class="row">
-        <h1 class="text-lg-center">Giving you a voice in the #OpenSource Tech community to inspire others</h1>
-        <div class="jumbotron">
+    <p></p>
```

Comment:
I initially did it to bring the paragraph down a bit since it was too close to the header.